lib/assignDisabledRanges.js

```diff
@@ -269,7 +269,8 @@ module.exports = function (root, result) {
 						createDisableRange(comment, start, false, description, end, false),
 					);
 				} else {
-					const range = _.last(disabledRanges[ALL_RULES]);
+					const ranges = disabledRanges[ALL_RULES];
+					const range = ranges[ranges.length - 1];
```

Comment:
[suggestion] [`_.last()`](https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L7609-L7612) is null-safe, so how about adding a null-check as below, just in case?

```suggestion
					const range = ranges == null ? null : ranges[ranges.length - 1];
```

Reply:
I added a few more null checks in ac07653d.